cluster/src/shard_tables_cache.rs

```diff
@@ -113,4 +156,82 @@ impl Inner {
         self.tables_by_shard
             .insert(tables_of_shard.shard_info.id, tables_of_shard);
     }
+
+    fn try_insert_table_to_shard(
+        &mut self,
+        prev_shard_version: u64,
```

Comment:
How about `type ShardVersion = u64`?

Reply:
Ok, I will define this alias in meta/type.rs.